database/schema.sql

```diff
@@ -1,88 +1,88 @@
+DROP TABLE IF EXISTS cancellation_types;
+DROP TABLE IF EXISTS services;
+DROP TABLE IF EXISTS operators;
+DROP TABLE IF EXISTS stations;
 
 
-CREATE TABLE Stations(
-    StationID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, 
-    CRSCode varchar(3) UNIQUE NOT NULL,
-    StationName varchar(60) NOT NULL
+CREATE TABLE stations (
+    station_id SERIAL PRIMARY KEY,
+    crs_code VARCHAR(3) UNIQUE NOT NULL,
+    station_name VARCHAR(60) NOT NULL
 );
 
-CREATE TABLE Operators(
-    OperatorID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, 
-    OperatorName varchar(60) NOT NULL,
-    OperatorCode varchar(2) UNIQUE
+CREATE TABLE operators (
+    operator_id SERIAL PRIMARY KEY,
+    operator_name VARCHAR(60) NOT NULL,
+    operator_code VARCHAR(2) UNIQUE
 );
 
-CREATE TABLE Services(
-    ServiceID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    ServiceUID varchar(6) UNIQUE NOT NULL,
-    OperatorID INT REFERENCES Operators(OperatorID)
+CREATE TABLE services (
+    service_id SERIAL PRIMARY KEY,
+    service_uid VARCHAR(6) UNIQUE NOT NULL,
+    operator_id INT REFERENCES operators(operator_id)
 );
 
-CREATE TABLE CancellationTypes(
-    CancellationTypeID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    CancellationCode varchar(2) UNIQUE NOT NULL,
-    Description text
+CREATE TABLE cancellation_types (
+    cancellation_type_id SERIAL PRIMARY KEY,
+    cancellation_code VARCHAR(2) UNIQUE NOT NULL,
+    description TEXT
 );
 
-CREATE TABLE Cancellations(
-    CancellationID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    ScheduledArrival timestamp NOT NULL,
-    CancellationTypeID INT REFERENCES CancellationTypes(CancellationTypeID),
-    StationID INT REFERENCES Stations(StationID),
-    ServiceID INT REFERENCES Services(ServiceID)
+CREATE TABLE cancellations (
+    cancellation_id SERIAL PRIMARY KEY,
+    scheduled_arrival TIMESTAMP NOT NULL,
+    cancellation_type_id INT REFERENCES cancellation_types(cancellation_type_id),
+    station_id INT REFERENCES stations(station_id),
+    service_id INT REFERENCES services(service_id)
 );
 
-CREATE TABLE Arrivals(
-    ArrivalID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    ScheduledArrival timestamp NOT NULL,
-    ActualArrival timestamp NOT NULL,
-    StationID INT REFERENCES Stations(StationID),
-    ServiceID INT REFERENCES Services(ServiceID)
+CREATE TABLE arrivals (
+    arrival_id SERIAL PRIMARY KEY,
+    scheduled_arrival TIMESTAMP NOT NULL,
+    actual_arrival TIMESTAMP NOT NULL,
+    station_id INT REFERENCES stations(station_id),
+    service_id INT REFERENCES services(service_id)
 );
 
-CREATE TABLE Users(
-    UserID INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    FirstName varchar(30) NOT NULL,
-    LastName varchar(60) NOT NULL,
-    PhoneNumber varchar(20),
-    Email varchar
+CREATE TABLE users (
+    user_id SERIAL PRIMARY KEY,
+    first_name VARCHAR(30) NOT NULL,
+    last_name VARCHAR(60) NOT NULL,
+    phone_number VARCHAR(20),
+    email VARCHAR
```

Comment:
Suggestion - can update to `email VARCHAR(320)` since the maximum length of an email address is 320 characters